### ISSUE TITLE
Improve post install message

### ISSUE THREAD

**Issue by philoserf** [Nordstrom/chefdk-julia#8]
If a chefdk-julia user follows the post install message and later uses a ruby (rvm, rbenv, chruby, etc.) that does not have chefdk-julia installed they will get an error message.

```
ERROR: You have an error in your config file [redacted]/.chef/knife.rb

LoadError: cannot load such file -- chefdk-julia
  [redacted]/.chef/knife.rb:1:in `from_string'
Relevant file content:
  1: require 'chefdk-julia'
  2: chefdk.generator_cookbook Chefdk::Julia.path
  3:
```

Perhaps the post install message could advice a begin/rescue block to allow fall back to the default generator.

**Comment by dougireton** [comment]
Thanks @philoserf. We'll take a look.

**Comment by dougireton** [comment]
Fixed by #9
